src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -379,3 +379,34 @@ However, it's preferable to use fenced code blocks over indented code blocks.
 Not only are fenced code blocks considered more idiomatic for Rust code,
 but there is no way to use directives such as `ignore` or `should_panic` with
 indented code blocks.
+
+### Include items only when collecting doctests
+
+Rustdoc's [documentation tests] can do some things that regular unit tests can't, so it can
```

Comment:
This link to `[documentation tests]` isn't necessary any more, since it's the same page that it's being written into.